firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/ApkUpdaterTest.java

```diff
@@ -170,6 +183,52 @@ public void updateApk_whenInstallSuccessful_setsResult() throws Exception {
     assertThat(updateTask.isSuccessful()).isTrue();
```

Comment:
[Not directly related to this PR, since this code was written beforehand] 

The test above (updateApk_whenInstallSuccessful_setsResult) seems like something we should remove. We can't set the result on a successful install because the process is killed

Reply:
Good point! Willing to delete it since I do not believe it adds to our current tests.